package.json

```diff
@@ -14,8 +14,7 @@
   "dependencies": {
     "css-vendor": "^0.3.1",
     "debug": "^2.1.3",
-    "lodash.throttle": "^3.0.3",
-    "object.assign": "^4.0.1"
```

Comment:
> Babel has its own implementation of Object.assign that is shared in the whole compiled code.

Hm, can you clarify please?

React Popover is compiled to ES5 before publishing so what users consume via NPM is ES5 code. When I use React Popover in e.g. a Babel-based JS2015 app that is _again_ compiled to ES5 before deployment **I fail to understand how Object.assign polyfill code is not simply duplicated**. Contrast with NPM which knows how to keep a flat dependency tree as of version 3.